source/agora/test/TimeDrift.d

```diff
@@ -40,29 +40,38 @@ unittest
 
     // sanity check for the generated quorum config
     nodes.enumerate.each!((idx, node) =>
-        retryFor(node.getQuorumConfig().threshold == 5 &&
-            node.getQuorumConfig().nodes.length == 5, 5.seconds,
-            format("Node %s has the wrong quorum config. Expecte 3/5. Got: %s",
+        retryFor(node.getQuorumConfig().threshold == conf.max_quorum_nodes &&
+                node.getQuorumConfig().nodes.length == conf.max_quorum_nodes,
+            5.seconds,
+            format("Node #%s has invalid quorum config for test: %s",
                 idx, node.getQuorumConfig())));
 
-    // set the time to `height` * `block_interval_sec` for 4/5 nodes
-    // clock times for nodes:    [ 1,  1,  1,  1,  0] median => 1
-    // calculated clock offset:  [+0, +0, +0, +0, +1]
-    network.setTimeFor(network.nodes.take(4), Height(1));
-    retryFor(nodes[0].getLocalTime() == 1 + network.genesis_start_time, 5.seconds);
-    retryFor(nodes[1].getLocalTime() == 1 + network.genesis_start_time, 5.seconds);
-    retryFor(nodes[2].getLocalTime() == 1 + network.genesis_start_time, 5.seconds);
-    retryFor(nodes[3].getLocalTime() == 1 + network.genesis_start_time, 5.seconds);
-    retryFor(nodes[4].getLocalTime() == 0 + network.genesis_start_time, 5.seconds);
+    // Check the node local time
+    void checkNodeLocalTime(ulong idx, ulong expected_height) {
```

Comment:
Style (brace and space before parenthese)

Reply:
Updated